app/models/energy/scheduleruleset.py

```diff
@@ -38,9 +47,13 @@ class ScheduleRule(BaseModel):
     """A set of rules assigned to schedule ruleset for specific periods of time and for
```

Comment:
I think we should also change the name of this class to `ScheduleRuleAbridged` if we are going to follow the convention that we have used throughout the schema.